dashboard/src/app/workspaces/list-workspaces/workspace-item/workspace-item.controller.ts

```diff
@@ -21,29 +25,47 @@ import {WorkspacesService} from '../../workspaces.service';
  */
 export class WorkspaceItemCtrl {
 
-  static $inject = ['$location', 'lodash', 'cheWorkspace', 'workspacesService'];
+  static $inject = ['$location', 'lodash', 'cheWorkspace', 'workspacesService', '$timeout', '$document', 'cheBranding', '$sce'];
 
   $location: ng.ILocationService;
   lodash: any;
   cheWorkspace: CheWorkspace;
   workspacesService: WorkspacesService;
+  $document: ng.IDocumentService;
+  $timeout: ng.ITimeoutService;
 
   workspace: che.IWorkspace;
   workspaceName: string;
-  workspaceSupportIssues = '';
+  workspaceSupportIssues: any;
+
+  private supportedRecipeTypeIssue: any;
+  private supportedVersionTypeIssue: any;
+  private timeoutPromise: ng.IPromise<any>;
 
   /**
    * Default constructor that is using resource
    */
   constructor($location: ng.ILocationService,
               lodash: any,
               cheWorkspace: CheWorkspace,
-              workspacesService: WorkspacesService) {
+              workspacesService: WorkspacesService,
+              $timeout: ng.ITimeoutService,
+              $document: ng.IDocumentService,
+              cheBranding: CheBranding,
+              $sce: ng.ISCEService) {
     this.$location = $location;
     this.lodash = lodash;
     this.cheWorkspace = cheWorkspace;
     this.workspacesService = workspacesService;
+    this.$timeout = $timeout;
+    this.$document = $document;
     this.workspaceName = this.cheWorkspace.getWorkspaceDataManager().getName(this.workspace);
+
+    this.supportedRecipeTypeIssue = $sce.trustAsHtml('Current infrastructure doesn\'t support this workspace recipe type.');
+
+    this.supportedVersionTypeIssue = $sce.trustAsHtml(`This workspace is using old definition format which is not compatible anymore. 
+          Please follow the <a href="${cheBranding.getDocs().workspace}" target="_blank">documentation</a>
```

Comment:
Would it be possible to reference the link to the doc section "Converting a Che 6 Workspace to a Che 7 devfile" ?

Reply:
Yes. It is possible. I have changed reference the link to the doc section "Converting a Che 6 Workspace to a Che 7 devfile"
https://github.com/eclipse/che/pull/13506/commits/2e2c2cc0aa78165f2b0b17da7b42ab370eb83f19

Reply:
done